README.md

```diff
@@ -29,15 +29,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+    map creates a new array and foreach mutates the array
```

Comment:
`The map() method creates a new array with the results of calling a function for every array element.`

`The forEach() method calls a provided function once for each element in an array, in order.`